docusaurus/docs/operate/morse_migration/overview.md

```diff
@@ -0,0 +1,248 @@
+
+2. **Facilitate offchain social consensus on MorseAccountState**:
+
+   - Using social consensus and cryptographic hash verification
+   - Offchain agreement (i.e. feedback loop) on the "canonical" `MorseAccountState`
+
+3. Minimize Shannon onchain state bloat
+
+   - Minimize the size & optimize performance of (Shannon) persisted onchain data
+   - Transform (offchain) the `MorseStateExport` into a `MorseAccountState`
+   - Persist minimal Morse account representations as individual `MorseClaimableAccount`s
+
+### ETVL High-Level Flow
+
+```mermaid
+flowchart
+
+  subgraph OncMorse[Morse On-Chain]
+    MorseState
+  end
+
+  MorseState --> | Export | MorseStateExport
+
+  subgraph OffC[Off-Chain]
+    MorseStateExport --> | Transform | MorseAppState
+    MorseAppState --> MorseApplications:::application
+    MorseAppState --> MorseAuth:::account
+    MorseAppState --> MorsePos:::supplier
+    MorseApplications:::application --> MorseApplication:::application
+    MorseAuth:::account --> MorseAuthAccount:::account
+    MorseAuthAccount:::account --> MAOff[MorseAccount]:::account
+    MorsePos:::supplier --> MorseValidator:::supplier
+  end
+
+
+  subgraph OnC[Shannon On-Chain]
+    MorseAccountState:::general
+    MorseAccountState:::general --> |Load| MAOn
+    MAOn[MorseClaimableAccount]:::general
+  end
+
+  MorseAppState --> |Validate| MorseAccountState
+
+
+  MAOff -.Track exported <br> account unstaked balance..-> MAOn
+  MorseValidator:::supplier -.Track exported <br> Supplier stake.-> MAOn
+  MorseApplication:::application -.Track exported <br> Application stake.-> MAOn
+
+  classDef account fill:#90EE90,color:#000
+  classDef supplier fill:#FFA500,color:#000
+  classDef application fill:#FFB6C6,color:#000
+  classDef general fill:#87CEEB,color:#000
+```
+
+### High-Level State Transport Playbook for Authority (i.e. Foundation)
+
+:::warning TODO_MAINNET: This playbook is a WIP
+
+This playbook is an early WIP and will be updated and moved elsewhere once the process is finalized.
+
+:::
+
+1. **Export** the canonical `MorseStateExport` from the Morse network:
+
+   ```bash
+   pocket util export-genesis-for-reset <published canonical export height> poktroll > morse_state_export.json
+   ```
+
+2. **Transform** the `MorseStateExport` into the proposed canonical `MorseAccountState`:
+
+   ```bash
+   poktrolld migrate collect-morse-accounts morse_state_export.json morse_account_state.json
+   ```
+
+3. **Distribute** the `MorseAccountState` and its hash for verification by Morse account/stake-holders.
+4. **Wait for consensus** after an offchain time-bounded period on the `MorseAccountState`, reacting to any feedback offchain as necessary.
+5. **Load** (i.e. import) the canonical `MorseAccountState` on Shannon
+
+   ```bash
+   TODO_MAINNET(@bryanchriswhite, #1034): Reference respective cmd or make target, once available.
+   ```
```

Comment:
:::important
The `--grpc-addr` flag expects the gRPC endpoint for the Shannon network you intend to import the Morse accounts into (e.g. DevNet, TestNet, etc.). See: [Shannon RPC Endpoints](/tools/endpoints/shannon_rpc.md) for more info.
:::